### PR TITLE
Cleanup code

### PR DESCRIPTION
**Pull request by tijsverkoyen**

### CODE REVIEW

**Comment by WouterSioen** [comment]
👍 on first sight

**Comment by tijsverkoyen** [comment]
I asked Grigory to review it

File: lib/forkcms_deploy/forkcms_3.7.rb [hunk 1]
```diff
@@ -1,9 +1,7 @@
-require "forkcms_deploy/forkcms_3"
+require "forkcms_deploy/forkcms_3.5"
 configuration = Capistrano::Configuration.respond_to?(:instance) ? Capistrano::Configuration.instance(:must_exist) : Capistrano.configuration(:must_exist)
 
 configuration.load do
-	set :shared_children, %w(files config install)
-
 	after 'deploy:update_code' do
 		composer.install_vendors
```
Comment (Extrapolator214) [inline lib/forkcms_deploy/forkcms_3.7.rb]: I think it can be removed as well, it's present in forkcms_3.4.rb

Comment (Extrapolator214) [inline lib/forkcms_deploy/forkcms_3.7.rb]: tasks composer:install_vendors and composer:install_composer are also in forkcms_3.4.rb

**Pull request merged**